mysql_ch_replicator/clickhouse_api.py

```diff
@@ -127,6 +158,16 @@ def execute_command(self, query):
                 if attempt == ClickhouseApi.MAX_RETRIES - 1:
                     raise e
                 time.sleep(ClickhouseApi.RETRY_INTERVAL)
+            except clickhouse_connect.driver.exceptions.DatabaseError as e:
```

Comment:
This is also not safe.
Table may already exists, but the table structure could change.
AFAIK this case is only possible when we apply migrations during initial replication, it's not so easy to fix as there could be alter queries / etc.
And we need some good tests for this logic if we wont to change it.